src/TelloRemoteControl/WifiConnection.cpp

```diff
@@ -2,13 +2,14 @@
 
 #include <WiFi.h>
 
+
 WifiConnection::WifiConnection(const WifiCredentials& creds) :
     _creds(creds)
 {
 
 }
 
-ConnectionStatus WifiConnection::connect(const uint64_t timeout_in_ms) const
+WifiConnection::ConnectionStatus WifiConnection::connect(const uint64_t timeout_in_ms) const
```

Comment:
Consider using `std::chrono::milliseconds` for `timeout` instead of calling it 'timeout_in_ms'